gen/templates/component/component-name.ads

```diff
@@ -269,6 +269,18 @@ package Component.{{ name }} is
    -- been updated.
    not overriding procedure Update_Parameters_Action (Self : in out Base_Instance) is abstract;
 
+   -- This function is called when the parameter operation type is "Validate". The default implementation of this
+   -- subprogram in the implementation package is a function that returns "Valid". However, this function can, and should be
+   -- overridden if something special needs to happen to further validate a parameter. Examples of this might be validation of
+   -- certain parameters beyond individual type ranges, or performing other special functionality that only needs to be
+   -- performed after parameters have been validated.
```

Comment:
Add that range checking is part of staging, so doesn't need to be implemented by the user in this function.

Reply:
Added in [c61b1f5](https://github.com/lasp/adamant/pull/64/commits/c61b1f53a99257b70b15bbd0a22495ca6cccd935).